base/complex.jl

```diff
@@ -1064,18 +1064,38 @@ end
 #Requires two different RoundingModes for the real and imaginary components
 """
     round(z::Complex[, RoundingModeReal, [RoundingModeImaginary]])
-    round(z::Complex[, RoundingModeReal, [RoundingModeImaginary]]; digits=, base=10)
-    round(z::Complex[, RoundingModeReal, [RoundingModeImaginary]]; sigdigits=, base=10)
+    round(z::Complex[, RoundingModeReal, [RoundingModeImaginary]]; digits=0, base=10)
+    round(z::Complex[, RoundingModeReal, [RoundingModeImaginary]]; sigdigits, base=10)
 
 Return the nearest integral value of the same type as the complex-valued `z` to `z`,
 breaking ties using the specified [`RoundingMode`](@ref)s. The first
 [`RoundingMode`](@ref) is used for rounding the real components while the
 second is used for rounding the imaginary components.
+                                                    
+If the `digits` keyword argument is provided, it rounds to the specified
+number of digits after the decimal place (or before if negative), in
+base `base`.
+
+If the `sigdigits` keyword argument is provided, it rounds to the
+specified number of significant `digits`, in base `base`.
+                                                    
```

Comment:
These keyword args are already documented in the default/non-Complex `round` docstring. I don't think repeating them is necessary since they have the same behavior.